backgroundScript.js

```diff
@@ -1,14 +1,20 @@
+// Listener for when the Chrome extension is installed.
 chrome.runtime.onInstalled.addListener(() => {
+	// Define the environment for the extension, either "prod" or "dev".
 	const environment = "prod";
 	// const environment = "dev";
+
+	// Set the website and backend URLs based on the environment.
 	const websiteUrl = (environment === "dev") ? "http://localhost:3000" : "https://vibinex.com";
 	const backendUrl = (environment === "dev") ? "http://localhost:8080" : "https://gcscruncsql-k7jns52mtq-el.a.run.app";
+
+	// Store the website and backend URLs in Chrome's local storage.
 	chrome.storage.local.set({ websiteUrl, backendUrl }).then(_ => console.log(`Website URL set to ${websiteUrl};`));
 
-	// API call to backend that created Rudderstack event
+	// Make an API call to the backend to create a Rudderstack event when the extension is installed.
 	chrome.storage.local.get(["userId"]).then(({ userId }) => {
 		const body = {
-			userId: userId ? userId : "anonymous-id",
+			userId: userId ? userId : "anonymous-id", // Use the stored userId or "anonymous-id" if not available.
```

Comment:
```suggestion
			userId: userId ? userId : "anonymous-id",
```

The comments are too much! The code is readable - comments are only making it harder to read the code itself.